README.md

```diff
@@ -1,19 +1,14 @@
 # WordPress for iOS #
 
-[![Build status](https://badge.buildkite.com/2f3fbb17bfbb5bba508efd80f1ea8d640db5ca2465a516a457.svg)](https://buildkite.com/automattic/wordpress-ios)
-[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
-
 ## Build Instructions
 
 Please refer to the sections below for more detailed information. The instructions assume the work is performed from a command line.
 
-> Please note – these setup instructions only apply to Intel-based machines. M1-based Mac support is coming, but isn't yet supported by our tooling.
-
 ### Getting Started
 
-1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. *WordPress for iOS* requires Xcode 11.2.1 or newer.
-1. From a command line, run `git clone git@github.com:wordpress-mobile/WordPress-iOS.git` in the folder of your preference.
-1. Now, run `cd WordPress-iOS` to enter the working directory.
+1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. *WordPress for iOS* requires Xcode `14.3.1` or newer.
+1. [Clone this repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) in the folder of your preference.
+1. In a terminal, enter the `WordPress-iOS` directory where it was cloned.
```

Comment:
Is this line necessary?

From one point of view, it's good not to make assumptions. Then again, it should be a given that once you clone a repo, you go into that folder to work on it..

Reply:
Good call, tweaked in https://github.com/wordpress-mobile/WordPress-iOS/pull/21825/commits/3e7ceeca8dbd2e2dbb04497628926659d221297c.

At the risk of being verbose I was thinking about the users who clone via GUI tools and needed to be in the terminal at that point. 👍